assembly/std/arguments.ts

```diff
@@ -198,7 +198,7 @@ export class Args {
    * @return {u32}
    */
   private toU32(byteArray: Uint8Array, offset: u8 = 0): u32 {
-    if (byteArray.length - offset < 8) {
+    if (byteArray.length - offset < 4) {
```

Comment:
I guess that you should clarify the meaning of that value ;-) 
```suggestion
    if (byteArray.length - offset < sizeof<u32>()) {
```